### DISCUSSION TITLE
Gmail marking email from support@npmjs.com as dangerous

### DISCUSSION THREAD

**Discussion by zeen** [community/community#145965] [npm]
My password reset email was in the spam folder, with a red warning at the top. Other emails may or may not be getting the same treatment. The admins may want to check their email deliverability stats to see if this is happening to all your emails.

Screenshot:

<img width="1125" alt="image" src="https://github.com/user-attachments/assets/84725b35-5026-4a15-8c6d-9ac83eb5703b">

**Comment by 0x4k-helios** [comment]
The admins should review their email deliverability stats, including SPF, DKIM, DMARC, and blacklist status, as the password reset email was flagged as spam with a warning.
